puma/plot_ftag.py

```diff
@@ -0,0 +1,288 @@
+from __future__ import annotations
+
+import argparse
+from pathlib import Path
+
+import numpy as np
+from ftag import Flavours
+
+from puma.utils import logger, set_log_level
+from puma.yuma import (
+    PlotConfig,
+    get_included_taggers,
+    get_plot_kwargs,
+    get_signals,
+    select_configs,
+)
+
+ALL_PLOTS = ["roc", "scan", "disc", "prob", "peff"]
+
+
+def get_args(args):
+    parser = argparse.ArgumentParser(description="YUMA: Plotting from Yaml in pUMA")
+    parser.add_argument("-c", "--config", type=str, help="Path to config")
+
+    parser.add_argument(
+        "-p",
+        "--plots",
+        nargs="+",
+        help=f"Plot types to make. Allowed are: {ALL_PLOTS} ",
+    )
+    parser.add_argument(
+        "-s",
+        "--signals",
+        nargs="+",
+        help="Signals to plot",
+    )
+    parser.add_argument(
+        "--sample",
+        default=None,
+        help="Sample to plot, by default all samples in config are plotted.",
+    )
+    parser.add_argument(
+        "-n",
+        "--num_jets",
+        required=False,
+        type=int,
+        help="Maximum number of jets to use in plotting, per model.",
+    )
+    parser.add_argument(
+        "-d", "--debug", action="store_true", help="Set logger level to debug."
+    )
+    return parser.parse_args(args)
```

Comment:
Are all these CLI args also exposed via the yaml config? Do we want/need to duplicate them?

If not would it not be more consistent to have everyting configured in the yaml?

Reply:
I've removed sample and debug, but I think its worth keeping the 'plots' and 'sample' as in a lot of cases the user might not want to make all plots.